server/core/auth.js

```diff
@@ -112,7 +113,8 @@ module.exports = {
       if (err) { return next() }
 
       // Expired but still valid within N days, just renew
-      if (info instanceof Error && info.name === 'TokenExpiredError' && moment().subtract(14, 'days').isBefore(info.expiredAt)) {
+      if (info instanceof Error && info.name === 'TokenExpiredError' &&
+        moment.subtract(ms(WIKI.config.auth.tokenRenewal), 'ms').isBefore(info.expiredAt)) {
```

Comment:
Should be `moment()`, otherwise you're relying on an existing instance which could have been modified previously.

Reply:
Ops my bad

Reply:
@NGPixel fixed, please review